operator/pkg/gateway-api/gatewayclass_status.go

```diff
@@ -4,24 +4,59 @@
 package gateway_api
 
 import (
+	"cmp"
+	"slices"
 	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	gatewayv1 "sigs.k8s.io/gateway-api/apis/v1"
+	"sigs.k8s.io/gateway-api/pkg/features"
 )
 
 const (
 	gatewayClassAcceptedMessage    = "Valid GatewayClass"
 	gatewayClassNotAcceptedMessage = "Invalid GatewayClass"
 )
 
+var supportedFeatures = features.AllFeatures
+
+var gatewayClassSupportedFeatures = getSupportedFeatures()
```

Comment:
assume the package level variable is used to avoid re-evaluating

Reply:
exactly 👍